lib/apiservers/service/swagger.json

```diff
@@ -798,6 +798,11 @@
               "type": "string"
             }
           }
+        },
+        "syslog_addr": {
+          "type": "string",
+          "format": "uri",
+          "pattern": "^(tc|ud)p.*"
```

Comment:
The documentation on `SysLogConfig` (in `container_vm.go`) says "Network can be udp, tcp, udp6, or tcp6". I think we should add the optional "6" to this regex.

Reply:
thanks, good catch

Reply:
https://github.com/vmware/vic/blob/master/lib/install/validate/validator.go#L860
Here in validator, only `udp` and `tcp` transport goes into config. It doesn't accept anything else
not sure if it's a bug

Reply:
confirmed with @hmahmood the ipv6 is not officially supported by syslogs yet. So we'll go with `^(tc|ud)p` for now